scenarios/23d_Trapped.cfg

```diff
@@ -36,9 +38,14 @@
             team_name=good
         [/modify_side]
         {VARIABLE ano_joined_23 yes}
-        {END_IF_WITHOUT_ELSE}
-        {END_IF_WITHOUT_ELSE}
+        {ELSE}
+        # TODO: test this branch
+        {MESSAGE (Raven O Trimmen) () (Raven O Trimmen) _"Really? I haven't heard anything about you being alive! You must be lying! I don't believe you!"}
         {END_IF}
+        {ELSE}
+        # TODO: test this branch
+        {MESSAGE (Raven O Trimmen) () (Raven O Trimmen) _"If you were truly King Gawen Hagarthen, you wouldn't have killed so many of my fellow Akladian Lords!"}
```

Comment:
I think just Gawen Hagarthen without the King since this guy is rude.

Reply:
done in [efba779](https://github.com/nemaara/A_New_Order/pull/43/commits/efba77972af8c9bcc1fc309bcc70c2a6748d2016)